aten/src/ATen/native/LinearAlgebra.cpp

```diff
@@ -48,14 +48,16 @@ namespace detail {
 namespace meta {
 
 #define ADDMM_META() \
+  TORCH_CHECK(self.scalar_type() == mat2.scalar_type(), "self and mat2 must have the same dtype"); \
+  TORCH_CHECK(mat1.scalar_type() == mat2.scalar_type(), "mat1 and mat2 must have the same dtype"); \
```

Comment:
It would be nice to add an ErrorInput to the addmm OpInfo test for this case

Reply:
I'm not sure what you mean by `ErrorInput` and I can't find any examples in the `test/` directory. Can you please provide an example of what you mean?